packages/engine/tests/simulations/state_edit_non_existing_field/integration-test.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
Should this test be about editing a `non-initialized` field instead? I think editing a non-existing field is something else, that we probably do want to test.

Reply:
To be very pedantic, I would just call it `edit_null_field`